media/js/src/filters/AssetFilter.jsx

```diff
@@ -215,6 +215,12 @@ export default class AssetFilter extends React.Component {
         const tagsOptions = tagsToReactSelect(this.props.allTags);
         const termsOptions = termsToReactSelect(this.props.allTerms);
 
+        const ownerFilterId = `ownerFilter-${(Math.random() + 1).toString(36).substring(7)}`;
+        const termFilterId = `termFilter-${(Math.random() + 1).toString(36).substring(7)}`;
+        const tagFilterId = `tagFilter-${(Math.random() + 1).toString(36).substring(7)}`;
+        const dateFilterId = `dateFilter-${(Math.random() + 1).toString(36).substring(7)}`;
+        const searchFilterId = `searchFilter-${(Math.random() + 1).toString(36).substring(7)}`;
```

Comment:
This repeated `(Math.random() + 1).toString(36).substring(7)` logic can definitely be refactored out into a common function here.